### PR TITLE
organizations.json and projects.json (partially) mock data

### PR DESCRIPTION
**Pull request by albannurkollari**
I created two mock-up data files. "organizations.json" and "projects.json" which consist of the requested properties in the "UserStories Overall Design.docx" file. It's no definitive but it's a start. Changes are welcome to come.

### CODE REVIEW

**Comment by IvanCybercom** [comment]
Please make the simplified files.

File: src/assets/mockdata/organizations.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
+{
+    "organizations": [
+        {
+            "id": 1,
+            "org_id": "ORG12-ISP9000-71812",
+            "name": "Org. A",
+            "address": [
+                {
+                    "label": "primary",
+                    "value": {
```
Comment (alayham) [inline src/assets/mockdata/organizations.json]: why do you nest the values?

File: src/assets/mockdata/organizations.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+                        "city": "Lund",
+                        "street": "Bonaröd 84",
+                        "postal": 28199
+                    }
+                }
+            ],
+            "contact": {
+                "phone": [
+                    {
```
Comment (alayham) [inline src/assets/mockdata/organizations.json]: I suggest a simple array without labels

File: src/assets/mockdata/organizations.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+                        "label": "primary",
+                        "value": "046-8807494"
+                    },
+                    {
+                        "label": "fax",
+                        "value": "0280-3418348"
+                    }
+                ],
+                "email": [
+                    {
```
Comment (alayham) [inline src/assets/mockdata/organizations.json]: I suggest a simple array without labels

File: src/assets/mockdata/organizations.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                    "Ivan",
+                    "Matts",
+                    "Oscar",
+                    "Reda"
+                ]
+            },
+            "password": "@#aaOB23WEpp!",
+            "billing": {
+                "address": [
+                    {
```
Comment (alayham) [inline src/assets/mockdata/organizations.json]: Did the product owner request multiple addresses?

**Comment by alayham** [comment]
No Comments. Ready to be Merged (RTBM)

**Pull request merged**